webhook/scripts/webhookLivenessProbe.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# if the livenessProbeSuccessOverride file is available, treat failures as success
+#
+RETVAL=$(test -f /webhook/debug-config/livenessProbeSuccessOverride ; echo $?)
```

Comment:
I'm wondering if these files need to be different... even if installed together, the operator and the webhook will be running in their own containers so they could use the same names for the .alive and other files.

Reply:
I changed the location of webhook liveness file to `/webhook/.alive` and the operator's liveness file is in `/operator/.alive` to avoid the confusion. If we decide to go with the same file, then both liveness files will need to be in the `/operator` directory.